how-we-work/all-hands-meeting.html.md

```diff
@@ -1,14 +1,39 @@
 ---
 title: All Hands meeting
 ---
-Every month, we hold an All Hands meeting. These meetings are used to:
-
-- give a general update on the status of Nebulab;
-- introduce new team members;
-- present any important updates to the playbook.
-
-You are strongly encouraged to attend so that you can ask questions and spend some virtual
-bonding time with the entire team, but in case you're not able to, the meetings are recorded.
+Every month, we hold a company-wide meeting, called the All Hands meeting. We use this
+session to provide useful information to every team member, with the intent of being as much
+transparent as possible about the direction of Nebulab, along with its successes, failures
+and next things to do. Having everyone on the same page is our way to try to drive every
```

Comment:
I'm not sure about this entire sentence, it just doesn't feel like it adds much value to your previous explanation of the meeting's intent.

Reply:
You are probably right, I'd just like to enforce that this meeting can help to reach our goals by sharing information and direction that can be used by all the team members day by day.